test/integration/main_test.go

```diff
@@ -1,5 +1,9 @@
 package main
 
+// Leaving this as an integration test artifact.
```

Comment:
How much effort will it take to fix?

Reply:
Tracking here: https://github.com/vmware/cluster-api-upgrade-tool/issues/84
This seems like it will be more involved.

Reply:
This will be a decently large effort. Probably a week or two to get these back in place.